backup/moodle2/backup_ratingallocate_activity_stepslib.php

```diff
@@ -69,25 +98,30 @@ protected function define_structure() {
         $ratingallocate_choice->add_child($groupchoices);
         $groupchoices->add_child($groupchoice);
 
-        // Define sources
+        $ratingallocate_choices->add_child($choicegroups);
```

Comment:
I think this should be `$ratingallocate_choice` instead as this corresponds to one choice.
Grouping should also be build in this tree.